pytorch_lightning/metrics/convertors.py

```diff
@@ -0,0 +1,207 @@
+import numbers
+from typing import Union, Any, Callable
+
+import numpy as np
+import torch
+from torch.utils.data._utils.collate import np_str_obj_array_pattern
+
+from pytorch_lightning.utilities.apply_func import apply_to_collection
+
+
```

Comment:
maybe a description on top of this file to let people know what it's doing?

Reply:
Done